Atomics.notify doesn't allow omitting last parameter

<!--
Please try to reproduce the issue with the latest published version. It may have already been fixed.
For npm: `typescript@next`
This is also the 'Nightly' version in the playground: http://www.typescriptlang.org/play/?ts=Nightly
-->
**TypeScript Version:**  3.8.3

<!-- Search terms you tried before logging this (so others can find this issue more easily) -->
**Search Terms:**
 - `Atomics.notify`
 - `Atomics`

**Code**

```ts
let sab = new SharedArrayBuffer(4);
let int = new Int32Array(sab);
Atomics.notify(int, 0);
```

**Expected behavior:**
Compiles successfully.

**Actual behavior:**
Getting an error:
```
../temp.ts:3:9 - error TS2554: Expected 3 arguments, but got 2.

3 Atomics.notify(int, 0);
          ~~~~~~~~~~~~~~

  ../../../AppData/Roaming/npm/node_modules/typescript/lib/lib.es2017.sharedmemory.d.ts:126:51
    126     notify(typedArray: Int32Array, index: number, count: number): number;
                                                          ~~~~~~~~~~~~~
    An argument for 'count' was not provided.


Found 1 error.
```

**Playground Link:** Actually, this is probably a separate issue, but playground seems to lack definitions for shared memory primitives, even though CLI does. E.g. check a link for the code above: [Playground Link](https://www.typescriptlang.org/play/index.html?target=99&ssl=1&ssc=1&pln=2&pc=22#code/DYUwLgBAzghgRhAvBAdiA7hAygCxgJxABMBBffGATwCEBXAM3pHwAoAWASgG4AoUSAJYpIyNJgCSwgMwAmMhUotYcbjxJgA9gFsBAYygA6FBrAD6ioWAA0EAAyqgA)

--- a/src/lib/es2017.sharedmemory.d.ts
+++ b/src/lib/es2017.sharedmemory.d.ts
@@ -102,8 +102,11 @@ interface Atomics {
     /**
      * Wakes up sleeping agents that are waiting on the given index of the array, returning the
      * number of agents that were awoken.
+     * @param typedArray A shared Int32Array.
+     * @param index The position in the typedArray to wake up on.
+     * @param count The number of sleeping agents to notify. Defaults to +Infinity.
      */
-    notify(typedArray: Int32Array, index: number, count: number): number;
+    notify(typedArray: Int32Array, index: number, count?: number): number;
 
     /**
      * Stores the bitwise XOR of a value with the value at the given position in the array,

